src/components/ShoppingList/styles.module.css

```diff
@@ -0,0 +1,40 @@
+.listElements {
+    width: 100%;
+}
+
+.newItem {
+    background-color: var(--item_grey);
+    color: var(--font_grey);
+    height: 40px;
+    border-radius: 20px;
+    margin-top: 10px;
+    box-sizing: border-box;
+    padding: 0 10px 0 28px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```

Comment:
The new item form and the item use a lot of the same css attributes, but are currently sourced from different files. Should this kind of thing be global? Or is it fine how it is?

Reply:
I think this is a sign that the wrapper at least should be the same component, this looks good for now though and we can refactor it later to re-use the same code more